samples/drivers/espi/src/main.c

```diff
@@ -317,26 +378,37 @@ int espi_test(void)
 #ifdef CONFIG_ESPI_GPIO_DEV_NEEDED
 	ret = wait_for_pin(gpio_dev0, CONFIG_PWRGD_PIN, PWR_SEQ_TIMEOUT, 1);
 	if (ret) {
-		printk("RSMRST_PWRGD timeout!");
+		LOG_ERR("RSMRST_PWRGD timeout!");
 		return ret;
 	}
 
 	ret = gpio_pin_set(gpio_dev1, CONFIG_ESPI_INIT_PIN, 1);
 	if (ret) {
-		printk("Failed to write %x %d\n", CONFIG_ESPI_INIT_PIN, ret);
+		LOG_ERR("Failed to write %x %d", CONFIG_ESPI_INIT_PIN, ret);
 		return ret;
 	}
 #endif
+	ret = wait_for_espi_reset(1);
+	if (ret) {
+		LOG_ERR("ESPI_RESET timeout!");
+		return ret;
+	}
 
 	ret = espi_handshake();
 	if (ret) {
-		LOG_PANIC();
+		LOG_ERR("eSPI handshake failed %d", ret);
 		return ret;
 	}
 
-	/* Attempt anyways to test failure case */
+	/*  Attempt anyways to test failure case */
 	get_pch_temp(espi_dev);
 
+	/* Cleanup */
+	k_sleep(1000);
```

Comment:
This will break once the new timeout API gets introduced. The more appropriate (and more readable) way that shouldn't break is `k_sleep(K_SECONDS(1));`

Reply:
Since I'm already updating this, updated.